.github/workflows/ci.yml

```diff
@@ -32,10 +27,16 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-yarn-
 
-     - uses: PaackEng/elm-validate-action@main
+     - uses: PaackEng/elm-validate-action@v1.1.0
+       with:
+         elm-make-options:
```

Comment:
do we need this empty param?

Reply:
Yes, to clear the default, as libs don't have/need `src/Main.elm --output=/dev/null`